src/views/cannot-add-user.njk

```diff
@@ -0,0 +1,30 @@
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% extends "layouts/default.njk" %}
+
+{% block back_link %}
+    {{ govukBackLink({
+        text: backLinkText,
+        href: backLinkHref
+    }) }}
+{% endblock %}
+
```

Comment:
You might be able to just import the nunjucks snippet for this if its a default back link with standard back text

{% block back_link %}
    {% include "partials/back_link.njk" %}
{% endblock %}